src/OSPSuite.Core/Domain/Mappers/MoleculeBuilderToMoleculeAmountMapper.cs

```diff
@@ -62,7 +76,11 @@ public IMoleculeAmount MapFrom(IMoleculeBuilder moleculeBuilder, IBuildConfigura
          //map parameters. Only parameters having BuildMode="Local" will
          //be added to the molecule amount. Global/Property-Parameters
          //will be filled in elsewhere (by the GlobalProperties-Mapper)
-         moleculeAmount.AddChildren(_parameterMapper.MapLocalFrom(moleculeBuilder, buildConfiguration));
+         var allLocalParameters = moleculeBuilder.Parameters
```

Comment:
Now I am making sure that the container where the molecule builder will be added is also fulfilling the container criteria if defined